docs/guides/custom-domain-name.md

```diff
@@ -50,6 +56,6 @@ AWS Gateway API Controller supports ExternalDNS integration through CRD source -
 
 ## Notes
 
-* You MUST have a registered hosted zone (e.g. `my-test.com`) in route53 and complete the `Prerequisites` mentioned in [Configure a custom domain name for your service](https://docs.aws.amazon.com/vpc-lattice/latest/ug/service-custom-domain-name.html#dns-associate-custom).
-* If you are not using ExternalDNS, you should manually associate your custom domain name with your service following [Configure a custom domain name for your service](https://docs.aws.amazon.com/vpc-lattice/latest/ug/service-custom-domain-name.html#dns-associate-custom).
+* You MUST have a registered hosted zone (e.g. `my-test.com`) in Route53 and complete the `Prerequisites` mentioned in [this section](https://docs.aws.amazon.com/vpc-lattice/latest/ug/service-custom-domain-name.html#dns-associate-custom) of the Amazon VPC Lattice documentation.
```

Comment:
I think the link should just be: ttps://docs.aws.amazon.com/vpc-lattice/latest/ug/service-custom-domain-name.html with no anchor. The "Prerequisites" section is toward the top.

Reply:
agree, changed